recipes/dotty-dict/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "dotty-dict" %}
+{% set version = "1.3.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: eb0035a3629ecd84397a68f1f42f1e94abd1c34577a19cd3eacad331ee7cbaf0
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+    - setuptools_scm
+  run:
+    - python >=3.5
+    - setuptools_scm
```

Comment:
Is `setuptools_scm` a run dependency? It seems like it would just be needed under `host`.

Reply:
I apologize. I forgot that `pip check` would need `setuptools_scm` here since it's under `install_requires`. The `setup.py` should probably only specify this under `setup_requires` since it's not a run time dep. You'll have to restore it to the `run` section.